.github/workflows/sync-project-fields-apply.yml

```
@@ -0,0 +1,82 @@
+name: Sync PR Project Fields - Apply
+
+on:
+  workflow_run:
+    workflows: ["Sync PR Project Fields - Compute"]
+    types: [completed]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.workflow_run.pull_requests[0].number || github.event.workflow_run.head_branch }
```

actionlint (1.7.12)

[error] got unexpected EOF while lexing end marker }}, expecting '}'

(expression)